us-medical-insurance-costs.ipynb

```diff
@@ -1365,9 +1367,71 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The Linear Regression Line is \n",
+      "Charges = 257.72 * Age + 3165.89\n",
+      "R is 0.299\n",
+      "Standard Error is 22.502\n"
+     ]
+    }
+   ],
+   "source": [
+    "# TheJJSerg Changes\n",
+    "# My Changes\n",
+    "\n",
+    "\n",
+    "\n",
+    "def simple_linear_regression(x, x_word, y = charges):\n",
+    "    \n",
+    "    # Define the following using linear regression\n",
+    "    slope, intercept, r, p, std_err = stats.linregress(x, y)\n",
+    "    \n",
+    "    # This function needs to be defined inside of the function for simple_linear_regression() to work\n",
+    "    def myfunc(x):\n",
+    "        return slope * x + intercept\n",
+    "    \n",
+    "    # Create a model\n",
+    "    model = list(map(myfunc, x))\n",
+    "    \n",
+    "    # Create a scatterplot and draw the linear regression line\n",
+    "    plt.scatter(x, y)\n",
+    "    plt.plot(x, model)\n",
+    "    \n",
+    "    # Label the graph\n",
+    "    plt.title(x_word.title() + \" Estimating Charges via Linear Regression\")\n",
+    "    plt.xlabel(x_word.title())\n",
+    "    plt.ylabel(\"Charges\")\n",
+    "    \n",
+    "    # Outputs\n",
+    "    plt.show() \n",
+    "    print(\"The Linear Regression Line is \")\n",
+    "    print(\"Charges = \" + str(round(slope, 2)) + \" * \" + str(x_word.title()) + \" + \" + str(round(intercept, 2)))\n",
+    "    print(\"R is \" + str(round(r, 3)))\n",
+    "    print(\"Standard Error is \" + str(round(std_err, 3)))\n",
+    "    return [slope, intercept, r, p, std_err]\n",
     "\n",
-    "# Team! We are limited by the constraints"
+    "simple_linear_regression_age = simple_linear_regression(x = age, x_word = \"age\")"
```

Comment:
Is it possible to extract the name directly from the list instead of specifying it?